ansible_builder/cli.py

```diff
@@ -43,6 +66,8 @@ def prepare(args=sys.argv[1:]):
             'After building the image, it can be used locally or published using the supplied tag.'
         )
     )
+
+    subparsers.add_parser('introspect')
     # TODO: Need to update the docstrings for the create and build commands to be more specific/helpful
```

Comment:
This `TODO` is left over from a previous PR; @AlanCoding changed the docstrings for those commands so this line can now be deleted.

Same with the line above that states `# TODO: Need to have a paragraph come up when running ansible-builder -h that explains what Builder is/does`